### ISSUE TITLE
Generate .vstags into the workspace storage

### ISSUE THREAD

**Issue by sogerc1** [vscode-perl/vscode-perl#40]
My project is mounted via sshfs, more people are working on it with different editors and it causes confusion when a .vstags appears in the project directory. Could you add an option to store the tags in ~/.config/Code/User/workspaceStorage/* like how the PHP intelephense extension does it?

**Comment by vctr-dev** [comment]
Hi @sogerc1, in https://github.com/vscode-perl/vscode-perl/blob/master/src/ctags.ts, you should be able to see that you could set the configuration `perl.ctagsFile` to whichever file you'd like to set it to.

**Comment by sogerc1** [comment]
Yes, there is indeed a configuration option to set the tags file but that is relative to the project directory and does not accept any ${something} variable (for example ${workspaceStorageDirectory}), so I can't place it into the workspace storage using that option. The only thing I can do is to set it to something like "../../../../../tmp/.vstags" but then the problem is that all projects use the same file and they constantly owerwrite it. I've also seen that I can have project specific settings but that causes the same problem I'm trying to avoid, a hidden directory appears in the project directory.
If only the tags file option could replace any project specific variable it would be good enough for me because then I could set it to something like "../../../../../tmp/${rootName}.vstags", that would be acceptable, the workspace storage directory would be ideal to me.
